app/assets/javascripts/discourse/app/lib/link-hashtags.js

```diff
@@ -0,0 +1,62 @@
+import { schedule } from "@ember/runloop";
+import { ajax } from "discourse/lib/ajax";
+import { replaceSpan } from "discourse/lib/category-hashtags";
+import { TAG_HASHTAG_POSTFIX } from "discourse/lib/tag-hashtags";
+
+const categoryHashtags = {};
+const tagHashtags = {};
+const checkedHashtags = [];
+
+export function linkSeenHashtags($elem) {
+  const $hashtags = $elem.find("span.hashtag");
+  if ($hashtags.length === 0) {
+    return [];
+  }
+
+  const slugs = $hashtags.map((_, hashtag) =>
+    $(hashtag)
+      .text()
+      .substr(1)
+  );
```

Comment:
Can probably be a one-liner

```suggestion
  const slugs = $hashtags.map((_, hashtag) =>$(hashtag).text().substr(1));
```

Reply:
I thought the same, but last time I checked, prettier did not agree.